source/hands_on/sqil.rst

```diff
@@ -39,14 +40,13 @@ Therefore, the value function is given by:
 
```

Comment:
In line 33, 'This is not' change with 'this is not'

Reply:
Always in line 33, sometimes you refer alpha as a 'parameter' while other times is referred as hyperparameter. By definition, parameters are learnable from the data while hyperparameters are manually tuned. Hence, it would be more correct to uniformly call 'alpha' a hyperparameter.